exiftool.yaml

```diff
@@ -0,0 +1,87 @@
+name: exiftool
+build-commands: []
+buildsystem: simple
+modules:
+  - name: exiv2
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DEXIV2_BUILD_EXIV2_COMMAND=OFF
+      - -DEXIV2_ENABLE_INIH=OFF
+      - -DEXIV2_BUILD_DOC:BOOL=OFF
+      - -DEXIV2_BUILD_SAMPLES=OFF
+    sources:
+      - type: archive
+        url: https://github.com/Exiv2/exiv2/archive/refs/tags/v0.28.3.tar.gz
+        sha256: 1315e17d454bf4da3cc0edb857b1d2c143670f3485b537d0f946d9ed31d87b70
+        x-checker-data:
+          type: anitya
+          project-id: 769
+          stable-only: true
+          url-template: https://github.com/Exiv2/exiv2/archive/refs/tags/v$version.tar.gz
+
+  - name: exiftool
```

Comment:
did you test the package? Perl is in the Sdk but NOT the platform.

Reply:
Yes and without exiftool rapid photo downloader don't work

Reply:
So how does it work?

```
$ flatpak run org.kde.Platform//5.15-23.08
[📦 org.kde.Platform ~]$ perl
sh: perl: command not found
```

tl;dr for Hugin I added Perl to the manifest to build it. Same for other packages that require Perl at runtime.

Reply:
never mind I missed it below.

Reply:
If you find a way to impove it without needed the perl build it could be cool but i dont know where to look

Reply:
it's all good. I just had missed perl that I was expecting to see before no inside.